assets/svelte/consumers/ShowHttpPull.svelte

```diff
@@ -143,32 +101,97 @@
     <div class="space-y-6">
       <Card>
         <CardContent class="p-6">
-          <h2 class="text-lg font-semibold mb-4">Configuration</h2>
-          <div class="grid md:grid-cols-2 gap-4">
-            <div>
-              <span class="text-sm text-gray-500">ID</span>
-              <p class="font-medium">{consumer.id}</p>
+          <div class="flex items-center space-x-4 mb-4">
+            <h2 class="text-lg font-semibold">Configuration</h2>
+            <div class="flex items-center">
+              <span
+                class="inline-flex items-center px-3 py-1 rounded-full text-xs font-medium bg-white border border-black text-black"
+              >
+                {#if "http_endpoint" in consumer}
+                  <ArrowRightToLine class="h-4 w-4 mr-1 text-black" />
+                  Push consumer
+                {:else}
+                  <ArrowLeftFromLine class="h-4 w-4 mr-1 text-black" />
+                  Pull consumer
+                {/if}
+              </span>
             </div>
-            <div>
-              <span class="text-sm text-gray-500">Message Kind</span>
-              <p class="font-medium">{consumer.message_kind}</p>
+            <div
+              class="inline-flex items-center px-3 py-1 rounded-full text-xs font-medium bg-white border border-black text-black"
+            >
+              <SquareStack class="h-4 w-4 mr-1 text-black" />
+              <p class="font-medium">
+                {#if consumer.message_kind === "event"}
+                  Change stream
+                {:else if consumer.message_kind === "record"}
+                  Sync stream
+                {:else}
+                  {consumer.message_kind}
+                {/if}
+              </p>
             </div>
+          </div>
+          <div class="grid md:grid-cols-2 gap-4">
             <div>
-              <span class="text-sm text-gray-500">Ack Wait</span>
-              <p class="font-medium">{consumer.ack_wait_ms} ms</p>
+              <span class="text-sm text-gray-500">Visibility Timeout</span>
+              <Tooltip.Root openDelay={300}>
+                <Tooltip.Trigger>
+                  <HelpCircle
+                    class="inline-block h-2.5 w-2.5 text-gray-400 -mt-2 cursor-help"
+                  />
+                </Tooltip.Trigger>
+                <Tooltip.Content class="max-w-xs">
+                  <p class="text-xs text-gray-500">
+                    <b>Visibility timeout:</b> Time limit for a message to be successfully
+                    acknowledged by the worker. Exceeding this time exposes the messages
+                    to other workers.
```

Comment:
newline, experiment with formatting, like this:

```suggestion
                    <b>Visibility timeout</b>
                    
                    <p>Time limit for a message to be successfully
                    acknowledged by the worker. After this time limit elapses, if the message has not yet been acknowledged, it will be available for delivery again to other workers.</p>
```

Reply:
not sure if `p` is right, play with it